src/OutputView.js

```diff
@@ -1,7 +1,53 @@
-export default OutputView = {
-    printMenu() {
-        Console.print("<주문 메뉴>");
-        // ...
+import { Console } from "@woowacourse/mission-utils";
+import { INFORMATION_MESSAGE, NOTHING_MESSAGE, TITLE_MESSAGE, ONE_CHAMPAGNE } from "./constants/index.js";
+
+const OutputView = Object.freeze({
+  // 기본 안내 메시지
+  greetingMessage() {
+    Console.print(INFORMATION_MESSAGE.GREETING);
+  },
+
+  eventMessage() {
+    Console.print(INFORMATION_MESSAGE.EVENT);
+  },
+
+  printMenus(orderMenu) {
+    Console.print(TITLE_MESSAGE.MENU);
+    Console.print(orderMenu.map(({ menu, count }) => `${menu} ${count}개`).join("\n"));
+  },
+
+  printPrice(title, price) {
+    Console.print(title);
+    Console.print(`${price.toLocaleString()}원`);
+  },
+
+  printGiftEvent(isGiftEvent) {
+    Console.print(TITLE_MESSAGE.GIFT);
+
+    if (isGiftEvent) {
+      Console.print(ONE_CHAMPAGNE);
+      return;
     }
```

Comment:
```suggestion
    if (isGiftEvent) return Console.print(ONE_CHAMPAGNE);
```
여기서 얼리 리턴하는거는 어떠실까요??

Reply:
기존에도 이미 `early return`을 사용하고 있기 한데, 말씀 해주신 방식은 `printGiftEvent` 메서드가 리턴하는 값이 생기기 때문에 목적에 맞지 않는 것 같아요! 

```js
printGiftEvent(); // 오직 이벤트를 출력하는 목적, 기존 방식
const giftEvent = printGiftEvent(); // 이벤트를 출력하지만 리턴 값이 있어 이렇게 오용할 수 있는 가능성이 생김
// giftEvent === Console.print(ONE_CHAMPAGNE)
```